api-gateway/src/main/java/com/petclinic/bffapigateway/presentationlayer/BFFApiGatewayController.java

```diff
@@ -57,31 +59,31 @@ public class BFFApiGatewayController {
     private final InventoryServiceClient inventoryServiceClient;
 
     @GetMapping(value = "bills/{billId}")
-    public Mono<BillDetails> getBillingInfo(final @PathVariable String billId)
+    public Mono<BillResponseDTO> getBillingInfo(final @PathVariable String billId)
     {
         return billServiceClient.getBilling(billId);
     }
 
     @PostMapping(value = "bills",
             consumes = "application/json",
             produces = "application/json")
-    public Mono<BillDetails> createBill(@RequestBody BillDetails model) {
+    public Mono<BillResponseDTO> createBill(@RequestBody BillRequestDTO model) {
         return billServiceClient.createBill(model);
     }
 
     @GetMapping(value = "bills")
```

Comment:
Fluxes should include produces TEXT_EVENT_STREAM_VALUE

Reply:
These are Monos